test/unit/language-test.js

```diff
@@ -337,6 +341,30 @@ describe('languageUtils', function() {
             expect(customLocalization.pause).to.equal(localizationPause);
             expect(customLocalization.stop).to.equal(localizationStop);
         });
+
+        it('should only use custom localization block if "forceLocalizationDefaults" is true', function() {
+            stubHtmlLanguage(document, 'fr');
```

Comment:
No need to stub the language on the hmtl, we can stub `getLanguage` to return 'fr'
import the entire language module `import * as Language from 'utils/language';`
`sandbox.stub(Language, 'getLanguage').returns('fr');`